tests/functional/u/unnecessary/unnecessary_dict_index_lookup.txt

```diff
@@ -1,23 +1,23 @@
-unnecessary-dict-index-lookup:7:10::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:11:10::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:17:36::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:19:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:20:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:20:44::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:22:33::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:24:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:25:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:25:41::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:34:10::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:35:21::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:39:40::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:41:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:42:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:42:52::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:44:37::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:46:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:47:1::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:47:49::Unnecessary dictionary index lookup, use 'v' instead
-unnecessary-dict-index-lookup:53:10::Unnecessary dictionary index lookup, use 'item[1]' instead
-unnecessary-dict-index-lookup:56:1::Unnecessary dictionary index lookup, use 'item[1]' instead
-unnecessary-dict-index-lookup:61:10::Unnecessary dictionary index lookup, use 'item[1]' instead
+unnecessary-dict-index-lookup:7:10::Unnecessary dictionary index lookup, use 'v' instead:HIGH
```

Comment:
```suggestion
unnecessary-dict-index-lookup:7:10::Unnecessary dictionary index lookup, use 'v' instead
```
Same here. It also has the benefit of a smaller diff which makes reviewing easier.

Reply:
The automated upgrade add the confidence now. I agree the diff is ruined by it. I think I should upgrade all of them at once so the diff is readable from now on.

Reply:
> I think I should upgrade all of them at once so the diff is readable from now on.

Should we though? I would argue it's perfectly fine the way it is now: only specify it if it changed.

Reply:
The aim was to make the test understandable in the first place and not have to handle a lot of complicated cases in test utils code. I think that if the output is always the same it helps to understand what the tests are doing even if it's often a HIGH value that is used and it's a little redundant. Could also have the benefit to force us to think if we're really highly confident that there is a problem, sometime we aren't.  Maybe we could modify the upgrade script so it does not add the HIGH if not required, but I think it will make the code hard to read if someone have to retro-engineer the functional tests to understand what is happening in them.

Reply:
Let's talk about it in another issue if required, I want to release ``2.9.2`` because I won't be available the end of this week and the false positive for ``unused-private-member`` when using ``self`` is important to fix fast imo.